scripts/run-tests.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2013 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import subprocess
+import sys
+
+import pytest # flake8: noqa
+import _pytest.main
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--junit', action='store_true', help='Output junit test report.')
+    parser.add_argument('--run-slow', action='store_true', help='Run slow tests.')
+    parser.add_argument(
+        'directories', nargs='+', help='Directories to run py.test on.')
+    args = parser.parse_args()
+
+    extra_args = []
+
+    if not args.run_slow:
+        extra_args.append('-m not slow and not flaky')
```

Comment:
Does this work? I would expect `-m` and `not slow and not flaky` to have to be separate elements in the list..